SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/service/SubscriptionService.java

```diff
@@ -227,6 +231,10 @@ private static void addProperty(Map<String, Object> resultMap, String key, Objec
 				propertyMap.put(NGSIConstants.PREVIOUS_VALUE_LIST,
 						((List<Map<String, Object>>) oldValue).get(0).get(NGSIConstants.NGSI_LD_HAS_LIST));
 			}
+			if (((List<Map<String, Object>>) oldValue).get(0).containsKey(NGSIConstants.NGSI_LD_HAS_JSON)) {
+				propertyMap.put(NGSIConstants.PREVIOUS_JSON,
+						((List<Map<String, Object>>) oldValue).get(0).get(NGSIConstants.NGSI_LD_HAS_JSON));
+			}
```

Comment:
Check notification functionality in case of previousJson.

Reply:
in notification previousJson working fine